pkg/ebpf/debugfs.go

```diff
@@ -0,0 +1,76 @@
+package ebpf
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/pkg/errors"
+)
+
+type kprobeStats struct {
+	hits int64
+	miss int64
+}
+
+// KPROBEPROFILE is the default path to the kprobe_profile file
+const KPROBEPROFILE = "/sys/kernel/debug/tracing/kprobe_profile"
```

Comment:
Is there any reason this is all caps? Usually go constants are not

Reply:
👍  I will rename it